Vladyslav_Gurevych/HW8/All.py

```diff
@@ -0,0 +1,18 @@
+def own_all(data: list):
```

Comment:
![image](https://user-images.githubusercontent.com/25909810/189629664-923075fa-88ea-4f64-ba1a-7b599561c6a8.png)
Try to avoid such naming

Reply:
fixed in all tasks